src/main/java/spike/command/ExitCommand.java

```diff
@@ -0,0 +1,20 @@
+package spike.command;
+
+import spike.task.TaskList;
+
+/**
+ * Handles data before exiting.
+ */
+public class ExitCommand extends Command {
+    public static final String EXIT_MESSAGE = "See you soon! ﾍ(=￣∇￣)ﾉ You may close this window now.";
+    /**
```

Comment:
You might want to put a line break to improve readability

Reply:
Thanks again for all the detailed inputs.